src/Raven.Server/Documents/Expiration/ExpiredDocumentsCleaner.cs

```diff
@@ -199,11 +199,11 @@ public void Dispose()
 
             if (SystemTime.UtcNow >= date)
                 throw new InvalidOperationException($"Cannot put an expired document. Expired on: {date.ToString("O")}");
-
-            var ticksBigEndian = Bits.SwapBytes((ulong)date.Ticks);
+            
+            var ticksBigEndian = IPAddress.HostToNetworkOrder(date.Ticks);
```

Comment:
Just noticed your fix: https://github.com/ayende/ravendb/commit/cd1449f2ced5 That explains everything.